site/en/docs/extensions/mv3/overview/index.md

```diff
@@ -6,30 +6,84 @@ updated: 2018-02-25
 description: An overview of the purpose of Chrome Extensions and how they're developed.
 ---
 
-Extensions are small software programs that customize the browsing experience. They enable users to
-tailor Chrome functionality and behavior to individual needs or preferences. They are built on web
-technologies such as HTML, JavaScript, and CSS.
+This page provides a very brief introduction to what extensions are and how you can make a trivial one.
 
-An extension must fulfill a [single purpose][1] that is narrowly defined and easy to understand. A
+
+## About extensions
+
+Extensions are small software programs that customize the browsing experience. They let users tailor
+Chrome functionality and behavior in many ways, providing things like:
+
+* Productivity tools
+* Web page content enrichment
+* Information aggregation
+* Fun and games
+
+These are just a few examples of the many things that extensions can do. See the [Chrome Web
+Store] to see thousands of different examples of published extensions.
+
+### How do extensions work?
+
+Extensions are built on web technologies such as HTML, JavaScript, and CSS.  They run in a separate,
+sandboxed execution environment and interact with the Chrome browser.  Consider the context for this
+-- Chrome runs on a device, connecting the user to resources on the Web, as depicted in this context
+diagram:
+
+{% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/5qVlqGn6vVMCWCSRclOY.png", alt="Context diagram of Chrome with an extension", width="517", height="476" %}
+
+Also shown in the diagram is an extension "plugged into" the Chrome browser. The extension can
+access and modify selected resources and actions in Chrome. There are two aspects to consider:
+
+**The extensions user interface**&emsp;This provides a consistent way for users to manage their
+extensions.
+
+**Extensions APIs**&emsp;The [extensions APIs](/docs/extensions/reference/) allow the extension's
+code to access features of the browser itself: activating tabs, modifying net requests, and so on.
+
+To create an extension, you assemble some resources -- a manifest, JavaScript and HTML files,
+images, and others -- that constitute the extension. For development and testing, you can load these
+"unpacked" into Chrome using extension developer mode. Once you are happy with your extension, you
+can package it and distribute it to users.
+
+### How do users get extensions?
+
+Most Chrome users get extensions from the Chrome Web Store (CWS). Developers across the globe
```

Comment:
```suggestion
Most Chrome users get extensions from the Chrome Web Store. Developers across the globe
```

We don't use CWS anywhere else in this article, or generally in the docs AFAIK.  It's not really a popular abbreviation outside of Google, I think.  (Doing a google search for "CWS" or even "CWS extension" doesn't really show much)

Reply:
Done.